std/special/compiler_rt/udivmod.zig

```diff
@@ -64,7 +65,7 @@ pub fn udivmod(comptime DoubleInt: type, a: DoubleInt, b: DoubleInt, maybe_rem:
         // K K
         // ---
         // K 0
-        if ((d[high] & (d[high] - 1)) == 0) {
```

Comment:
I think in this case it wouldn't cause any harm, but just as a general rule, there are some reasons to avoid calling std lib functions in compiler-rt:
 * Technically, all zig code has a dependency on compiler-rt. But compiler-rt implementations have been coded carefully to avoid circular dependencies. Generally avoiding std lib calls helps to make this simple.
 * Most compiler-rt functions are a single function call, and don't put more than 1 frame on the stack. I'm not sure if this has any semantic meaning or not, but it's a property I noticed and I'd be careful before disturbing it.
 * Compiler-rt implementations are ported from llvm's compiler-rt project. So maintaining code that looks pretty close to the original is beneficial when trying to merge upstream changes. Zig's compiler-rt code is a *maintained port*.
 * Note the existing use of std was comptime only for the Log2Int type.

Reply:
Makes sense--reverted the changes to that file.